content/_data/changelogs/weekly.yml

```diff
@@ -15194,6 +15194,203 @@
   # pull: 6453 (PR title: Improve queue restart tests)
   # pull: 6454 (PR title: Remove unnecessary shutdown hook)
 
+  - version: '2.344'
+    date: 2022-04-17
+    changes:
+      - type: TODO
+        category: developer
+        pull: 6480
+        issue: 68276
+        authors:
+          - jtnord
+        pr_title: "[JENKINS-68276] Upgrade bundled Jackson 2 API plugin from 2.12.0\
+          \ to 2.13.2.20220328-273.v11d70a_b_a_1a_52"
+        message: |-
+          Upgrade bundled Jackson 2 API plugin from 2.12.0 to 2.13.2.20220328-273.v11d70a_b_a_1a_52.
+      - type: TODO
+        category: developer
+        pull: 6474
+        authors:
+          - dependabot[bot]
+        pr_title: Upgrade Spring Framework from 5.3.18 to 5.3.19
+        message: |-
+          Upgrade [Spring Framework](https://spring.io/projects/spring-framework) from 5.3.18 (released on March 31, 2022) to 5.3.19 (released on April 13, 2022). Spring Framework 5.3.19 includes [12 fixes and improvements](https://github.com/spring-projects/spring-framework/releases/tag/v5.3.19). In addition, Spring Framework 5.3.19 includes a fix for [CVE-2022-22968: Spring Framework Data Binding Rules Vulnerability](https://spring.io/blog/2022/04/13/spring-framework-data-binding-rules-vulnerability-cve-2022-22968).
```

Comment:
The hyperlinks in this comment will need to be converted to references after the weekly release is built and tagged.